api/v1beta2/ibmpowervsmachine_webhook.go

```diff
@@ -136,15 +126,15 @@ func (r *IBMPowerVSMachine) validateIBMPowerVSMachineImage() *field.Error {
 }
 
 func (r *IBMPowerVSMachine) validateIBMPowerVSMachineMemory() *field.Error {
-	if res := validateIBMPowerVSMemoryValues(r.Spec.Memory); !res {
```

Comment:
wondering if we can just stick to current default which is 2GB

Reply:
Fixed.